### PR TITLE
Clarifying displayed name

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6274

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-call-queue.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ To set up a call queue, in the Teams admin center, expand **Voice**, click **Cal
 
 ![Screenshot of resource account and language settings](media/call-queue-name-language.png)
 
-1. Type a name for the call queue. Agents will see this name when they receive an incoming call from the queue.
+1. Type a name for the call queue. (Please note that the name the Agents will see when they receive an incoming call will be the Resource account name).
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:50]:
```suggestion
1. Type a name for the call queue. (Please note that the name the agents will see when they receive an incoming call will be the resource account name.)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**